codeChallenge2.js

```diff
@@ -0,0 +1,18 @@
+var amounts = [124, 48, 268];
+
+function addTip(sum){
```

Comment:
```suggestion
const addTip = sum => { 
```

If you wanna be a cool JS-kiddo you define it like this (the functionality is the same)

Reply:
I do wanna be a cool JS-kiddo. So those are arrow functions, new in ES6 (js version 2015) to shorten the syntax and remove the 'this'. Although I don't know yet what 'this' means :P unless it's similar to constructors in python

Reply:
I just got onto 'this' in today's lecture. It is very like python in the main concept.
var john = {
    firstname: 'John',
    birthYear: 1994,
    family: ['Jane', 'Mark'],
    isMarried: false,
    calcAge: function(){
        this.age = 2018 - this.birthYear;           //this
    }
}